Formula/liquidprompt.rb

```diff
@@ -17,13 +17,6 @@ def caveats; <<-EOS.undent
       if [ -f #{HOMEBREW_PREFIX}/share/liquidprompt ]; then
         . #{HOMEBREW_PREFIX}/share/liquidprompt
       fi
-
-    If you'd like to reconfigure options, you may do so in ~/.liquidpromptrc.
-    A sample file you may copy and modify has been installed to
-      #{HOMEBREW_PREFIX}/share/liquidpromptrc-dist
```

Comment:
I don't think it's worth keeping — that sample file will be available online for people that want it, and I can't see why it would ever be important to anybody where Homebrew kept its copy of it. Honestly, there's probably not a lot of point in Homebrew even installing it.